deploy/kubernetes/auditbeat-kubernetes.yaml

```diff
@@ -16,6 +16,17 @@ data:
 
     processors:
       - add_cloud_metadata:
+      - add_process_metadata:
+          match_pids: ['process.pid']
+          include_fields: ['container.id']
+      - add_kubernetes_metadata:
+          host: ${NODE_NAME}
+          default_indexers.enabled: false
+          default_matchers.enabled: false
+          indexers:
+            - container:
+          matchers:
+            - fields.lookup_fields: ['container.id']
```

Comment:
We could probably make these indexers and matchers the default ones for auditbeat.

Reply:
++ on doing this

Reply:
Issue created for this: https://github.com/elastic/beats/issues/17432